core/Session/SessionAuth.php

```diff
@@ -186,4 +192,24 @@ private function updateSessionExpireTime()
         setcookie(session_name(), session_id(), time() + $sessionCookieLifetime, $sessionParams['path'],
             $sessionParams['domain'], $sessionParams['secure'], $sessionParams['httponly']);
     }
+
+    private function isExpiredSession(SessionFingerprint $sessionFingerprint)
+    {
+        $nonRememberedSessionExpireTime = Config::getInstance()->General['login_session_not_remembered_idle_timeout'];
+        $sessionCookieLifetime = Config::getInstance()->General['login_cookie_expire'];
```

Comment:
is the check for `login_cookie_expire`  actually needed or already done somewhere else? Didn't even know we keep the session only active for 14 days.

Reply:
I see... we're extending the session for 14 days each time after it was used.